themes/default/content/blog/pulumiup-google-native-provider/index.md

```diff
@@ -42,62 +42,74 @@ Google Cloud Native is available in public preview today for all Pulumi language
 {{% choosable language typescript %}}
 
 ```typescript
-import * as storage from "@pulumi/google-native/storage/v1";
 import * as pulumi from "@pulumi/pulumi";
-const config = new pulumi.Config("google-native");
-const project = config.require("project");
-const bucketName = "pulumi-google-native-ts-01";
-// Create a Google Cloud resource (Storage Bucket)
-const bucket = new storage.Bucket("my-bucket", {
-    name:bucketName,
-    bucket:bucketName,
-    project: project,
+import * as google from "@pulumi/google-native";
+
+const project = "mythical-ember-311422";
+const region = "us-central1";
```

Comment:
It might be nice to pull these from config so that they don't look hard-coded.

Reply:
Ah, I debated doing that! I selected this option so that one could easily copy-paste this example from the blog into a Pulumi program. But, if our precedent is to use config, I'll adjust.